Taskfile.yml

```diff
@@ -20,11 +23,18 @@ tasks:
     cmds:
       - task: build
       - mkdir -p {{.BIN_LOCATION}}
+      - mkdir -p {{.HOME}}/.local/state/ron-mod-manager
```

Comment:
Kinda sucks to have to make these dirs on install. Feel like I gotta move this stuff into the runtime

Reply:
Yeah, I'd highly suggest just running [`os.MkdirAll`](https://pkg.go.dev/os#MkdirAll)